go-apps/meep-platform-ctrl/server/user_authentication.go

```diff
@@ -58,74 +274,54 @@ func uaLoginUser(w http.ResponseWriter, r *http.Request) {
 		count := sessionStore.GetCount()
```

Comment:
Verifying count should be the first step in the `startSession` function

Reply:
It is the first thing checked in the case where we need to create a new session.